packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -120,33 +105,24 @@ Chunking is relevant for both `FlushMode.TurnBased` and `FlushMode.Immediate` as
 
 ## Disabling in case of emergency
 
-If the features are enabled using the configs, they can be disabled at runtime via feature gates as following:
+Compression and Chunking configuration can be overridden via feature gates to force-disable them:
 
 -   `Fluid.ContainerRuntime.CompressionDisabled` - if set to true, will disable compression (this has a side effect of also disabling chunking, as chunking is invoked only for compressed payloads).
--   `Fluid.ContainerRuntime.DisableGroupedBatching` - if set to true, will disable grouped batching.
 -   `Fluid.ContainerRuntime.CompressionChunkingDisabled` - if set to true, will disable chunking for compression.
 
 ## Example configs
 
-By default, the runtime is configured with the following values related to compression and chunking:
-
-```
-    const runtimeOptions: IContainerRuntimeOptions = {
-        compressionOptions: {
-            minimumBatchSizeInBytes: 614400,
-            compressionAlgorithm: CompressionAlgorithms.lz4,
-        },
-        chunkSizeInBytes: 204800,
-        maxBatchSizeInBytes: 716800,
-    }
-```
-
-To enable grouped batching, use the following property:
+By default, the runtime is [configured](..\containerRuntime.ts#L572) with the following values related to compression and chunking:
```

Comment:
I'd not copy config here, but rather make a reference to a code - this will become obsolete very quickly.

Reply:
Yeah, there's a link to the code. Good point about falling out of sync, I'll remove the copy.